litellm/main.py

```diff
@@ -65,6 +65,31 @@ def completion(
     # Optional liteLLM function params
     *, force_timeout=60, azure=False, logger_fn=None, verbose=False
   ):
+  # Docstring
```

Comment:
I love the idea of a doc string, however I'd rather users immediately see the code of calling completions. 

When I see the doc string it seems slightly overwhelming here. 

Perhaps we can add this to https://litellm.readthedocs.io/en/latest/

Reply:
Deleted and saw everything in /input.md 🥲
Cheers!